ios/Classes/PspdfkitPlugin.m

```diff
@@ -7,6 +7,8 @@
 //  This notice may not be removed from this file.
 //
 #import "PspdfkitPlugin.h"
+#import "PspdfFlutterHelper.h"
```

Comment:
```suggestion
```

This causes a compile error

Reply:
Aah, my bad. I'll remove this - I was trying to clean up some code.